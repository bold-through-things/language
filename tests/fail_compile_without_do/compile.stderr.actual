[
  {
    "recoverable": false,
    "line": 2,
    "char": 0,
    "content": "for i in",
    "error": "failed to assert: must have a `do` block",
    "error_type": "ASSERTION_FAILED"
  },
  {
    "recoverable": false,
    "line": 12,
    "char": 0,
    "content": "fn name",
    "error": "failed to assert: function must have a do block",
    "error_type": "ASSERTION_FAILED"
  },
  {
    "recoverable": false,
    "line": 9,
    "char": 0,
    "content": "while",
    "error": "failed to assert: must have two children, got [\"true\"]",
    "error_type": "ASSERTION_FAILED"
  }
]
