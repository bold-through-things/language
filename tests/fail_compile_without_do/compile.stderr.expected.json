{
  "$items": [
    {
      "recoverable": false,
      "line": 2,
      "char": 0,
      "content": { "$contains": "for i in" },
      "error_type": "ASSERTION_FAILED"
    },
    {
      "recoverable": false,
      "line": 12,
      "char": 0,
      "content": { "$contains": "fn name" },
      "error_type": "ASSERTION_FAILED"
    },
    {
      "recoverable": false,
      "line": 9,
      "char": 0,
      "content": { "$contains": "while" },
      "error_type": "ASSERTION_FAILED"
    }
  ],
  "$order": "any"
}
